### PR TITLE
Golangci lint v1.43.0 changes

### PR DESCRIPTION
**Pull request by ethanadams**
What: 
Changes to fix or ignore errors returned by golangci-lint v1.43.0
- dupArg: suspicious method call with the same argument and receiver (gocritic)
- File is not `gofmt`-ed with `-s` (gofmt)

Why:
- We want to upgrade golangci-lint
- 
Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [x] NEW: Are there any Satellite database migrations? Are they forwards _and_ backwards compatible? 
 - [x] Does the PR describe what changes are being made?
 - [x] Does the PR describe why the changes are being made?
 - [x] Does the code follow [our style guide](https://github.com/storj/docs/blob/main/code/Style.md)?
 - [x] Does the code follow [our testing guide](https://github.com/storj/docs/blob/main/code/Testing.md)?
 - [x] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [x] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [x] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [x] Does any documentation need updating?
 - [x] Do the database access patterns make sense?

### CODE REVIEW

File: fpath/path_unix_test.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-// +build linux darwin
```
Comment (egonelbre) [inline fpath/path_unix_test.go:4]: This old version must stay in place for old Go versions.

File: storj/node_test.go [hunk 2]
```diff
@@ -146,13 +146,13 @@ func TestNodeID_String_Version(t *testing.T) {
...
-	require.Equal(t, 0, a.Compare(a))
+	require.Equal(t, 0, a.Compare(a)) //nolint: gocritic
 
 	for k := 0; k < len(storj.NodeID{}); k++ {
 		var a, b storj.NodeID
 		a[k], b[k] = 1, 2
-		require.Equal(t, 0, a.Compare(a))
-		require.Equal(t, 0, b.Compare(b))
+		require.Equal(t, 0, a.Compare(a)) //nolint: gocritic
+		require.Equal(t, 0, b.Compare(b)) //nolint: gocritic
```
Comment (egonelbre) [inline storj/node_test.go:155]: Is there a way to specific a specific reason instead of disabling the whole linter?

**Review by egonelbre — CHANGES_REQUESTED** [review]

**Comment by egonelbre** [comment]
Also based on the comments, you shouldn't need to update them manually. You can use `go fmt ./...` to fix them automatically.

**Review by egonelbre — APPROVED** [review]

**Review by stefanbenten — APPROVED** [review]

**Comment by egonelbre** [comment]
run jenkins

**Comment by egonelbre** [comment]
run jenkins

**Pull request merged**